game-core/src/main/java/games/strategy/triplea/ResourceLoader.java

```diff
@@ -89,26 +83,6 @@ public ResourceLoader(final String mapName) {
     this.mapName = mapName;
   }
 
-  /**
-   * Will return an empty string unless a special prefix is needed, in which case that prefix is
-   * constructed based on where the {@code baseTiles} folder is located within the zip.
-   */
-  private static String getMapPrefix(final File mapZip) {
-    try (ZipFile zip = new ZipFile(mapZip)) {
-      final Optional<? extends ZipEntry> baseTilesEntry =
-          zip.stream()
-              .filter(entry -> entry.getName().endsWith(REQUIRED_ASSET_EXAMPLE_FOLDER))
-              .findAny();
-      if (baseTilesEntry.isPresent()) {
-        final String path = baseTilesEntry.get().getName();
-        return path.substring(0, path.length() - REQUIRED_ASSET_EXAMPLE_FOLDER.length());
```

Comment:
@jdimeo , this update now has everything being a folder. Does the submodule structure you have still work as directories rather than as expanded zip files?

Reply:
@DanVanAtta I'm not sure! I think this discovery process may still have to be re implemented for the folders, but I also know that the unzipped maps had less hardcoded assumptions about where things were located.

Reply:
Thanks. It's certainly a bit of a mess that TripleA assumes that a map name is equal to the map folder. This is something I'm trying to remove. Once the 'map.yml' file lands, it might be a lot of what you would want in order to have more customized locations. Regardless, that would probably be the right time FWIW to 'fix' the path discovery.